package.json

```diff
@@ -128,6 +128,7 @@
     "dependencies": {
         "@deriv/quill-icons": "^1.19.8",
         "@headlessui/react": "^1.7.18",
-        "usehooks-ts": "^3.0.1"
+        "usehooks-ts": "^3.0.1",
```

Comment:
```suggestion
```
i couldn't find 'usehooks-ts' usage anywhere in the PR :)

Reply:
Yes, correct, I'm not using this package, it was added before me. I just insert comma in order to add react-swipeable library to package.json

Reply:
oh)) ok ok)